hydroshare/settings.py

```diff
@@ -7,6 +7,9 @@
 import sys
 # import importlib
 
+# detect test mode to turn off some features
+TESTING = len(sys.argv) > 1 and sys.argv[1] == 'test'
```

Comment:
I do not know anything about this TESTING setting.  Is it used anywhere else?  It looks like @alvacouch set it up 4 years ago.

Reply:
Scott you're right that @alvacouch added this, I just moved it up higher in settings.py so that I could also use it as recommended here:
https://docs.djangoproject.com/en/1.11/ref/contrib/staticfiles/#django.contrib.staticfiles.storage.ManifestStaticFilesStorage.manifest_strict

> During testing, ensure that the [STATICFILES_STORAGE](https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-STATICFILES_STORAGE) setting is set to something else like 'django.contrib.staticfiles.storage.StaticFilesStorage' (the default).